classes/calendar.py

```diff
@@ -0,0 +1,9 @@
+class Calender():
+  def __init__ (self, id, calender_posts, text, date):
+    self._id = id
+    self._calender_posts = calender_posts
+    self._text = text
+    self._date = date
+    return self
+  
```

Comment:
Did we lose the `calendar_posts` class?

Reply:
yes, did you not request for it to be an attribute?